src/EditorFeatures/CSharpTest/NavigateTo/NavigateToTests.cs

```diff
@@ -43,7 +46,7 @@ protected override TestWorkspace CreateWorkspace(string content, ExportProvider
 }", async w =>
             {
                 var item = (await _aggregator.GetItemsAsync("Goo")).Single(x => x.Kind != "Method");
-                VerifyNavigateToResultItem(item, "Goo", "[|Goo|]", MatchKind.Exact, NavigateToItemKind.Class, Glyph.ClassInternal);
```

Comment:
Can you now remove "#pragma warning disable CS0618 // MatchKind is obsolete" from this file too?